engine/reference/commandline/service_scale.md

```diff
@@ -25,8 +25,8 @@ Options:
 
 ### Scale a service
 
-If you scale a service, you set the *desired* number of replicas. Even though
-the command returns directly, actual scaling of the service may take some time.
+Scales a service both up and down to the desired number of replicas. Even though the command returns directly, actual scaling of the service may take some time. To keep a service active but have no instances of it you can scale it to 0.
```

Comment:
Change "both up and down" to "either up or down" as you really can't do both at the same time.

Maybe reword the second part to something like "The command returns immediately, but the service may take some time to finish scaling."

Maybe "You can set a service's scale to 0 to keep the service active but running zero instances." Maybe say why you would do that, too.